### DISCUSSION TITLE
Actionsheet and select not working

### DISCUSSION THREAD

**Discussion by mbegin** [gluestack/gluestack-ui#1235] [Q&A]
Hello,

Since upgrading gluestack-ui to the newest version, the actionsheet and select components have stopped working for me. The select component was working prior to upgrading to the themed package. I am using expo sdk 48 and running android emulator. I also tried the examples on the documentation but clicking on the button does not open the dropdown. Any ideas what i should try to get this working?

**Comment by DennisHesler** [comment]
hi, did you find the solution? 
I have This problem

**Comment by Viraj-10** [comment]
@mbegin @DennisHesler Can you create a issue for this? Please upgrade to latest version if it doesn't work. Please report a detailed issue and code.
